src/helix-ui/elements/HXFileInputElement.less

```diff
@@ -0,0 +1,41 @@
+@import './HXElement';
+@import 'elements/hx-icon';
+@import (reference) "components/buttons";
+
+:host {
+  #hxContent {
+    display: none;
```

Comment:
Document that this is needed to prevent focusing of native file inputs. Especially true for Microsoft browsers, because Edge and IE have double tab stops.